src/components/modules/Earn/SpendCardCollection/SpendCardCollection.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import SpendCard from "../SpendCard";
+import styles from "./SpendCardCollection.module.css";
+
+// Get content data
+let rewards = require("../../../../content/collections/rewards.json").data;
+
+export default function SpendCardCollection() {
+  return (
+    <div className={styles.collection}>
+      {// rendering our prizes
+      rewards.prizes.map((entry, i) => {
+        return <SpendCard entry={entry} />;
```

Comment:
This might be a nit, but passing in the full `entry` object here feels weird to me. Explicitly declaring the props, or using the spread operator, would make the code a bit more approachable